engine/src/main/java/io/camunda/zeebe/engine/processing/bpmn/container/MultiInstanceBodyProcessor.java

```diff
@@ -37,10 +37,21 @@
 
   private static final DirectBuffer NIL_VALUE = new UnsafeBuffer(MsgPackHelper.NIL);
   private static final DirectBuffer LOOP_COUNTER_VARIABLE = BufferUtil.wrapString("loopCounter");
+  private static final DirectBuffer NUMBER_OF_ELEMENT_INSTANCES_VARIABLE =
+      BufferUtil.wrapString("numberOfElementInstances");
+  private static final DirectBuffer NUMBER_OF_ACTIVE_ELEMENT_INSTANCES_VARIABLE =
+      BufferUtil.wrapString("numberOfActiveElementInstances");
```

Comment:
❌  Let's use the same naming as defined in the spec: `numberOfInstances` and `numberOfActiveInstances`, respectively. These are also used by Camunda Platform 7.